### PR TITLE
Added address autocomplete and updated db

### PR DESCRIPTION
**Pull request by marat-davudov**
Updated the business form to do autocomplete and geocoding of the address using Google Places API. 

For this to work correctly, you need to add the NEXT_PUBLIC_GOOGLE_MAPS_API_KEY variable into your .env.local file.

I have also modified the schema.ts to reflect additional stored information: address fields and lat/lng.

### CODE REVIEW

File: components/forms/business-form/business-form.tsx [hunk 1]
```diff
@@ -30,8 +38,41 @@ export const BusinessForm: React.FC<BusinessFormProps> = ({
...
+    city: initialData?.city || "",
+    state: initialData?.state || "",
+    latitude: initialData?.latitude || null,
+    longitude: initialData?.longitude || null,
   });
 
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (!isLoaded || loadError) return;
```
Comment (AbbasJay) [inline components/forms/business-form/business-form.tsx]: i would maybe just split these and if there is a loadError we can just log the loadError in the console

**Comment by AbbasJay** [comment]
everything looks good, you will get a few merge conflicts if my PR gets merged in first

File: components/forms/business-form/business-form.tsx (hunk 1 above)
Comment (marat-davudov) [reply components/forms/business-form/business-form.tsx]: I've split them with the latest commit, is that what you were looking for?

Comment (AbbasJay) [reply components/forms/business-form/business-form.tsx]: Yes, perfect

File: lib/db/schema.ts [hunk 2]
```diff
@@ -65,6 +72,10 @@ export const ClassesTable = pgTable(
     capacity: integer("capacity").notNull(),
     slotsLeft: integer("slots_left").notNull(),
     createdAt: timestamp("created_at").defaultNow().notNull(),
+    startDate: text("start_date").notNull(),
+    time: text("time").notNull(),
+    slotsLeft: integer("slots_left").notNull(),
+    location: text("location").notNull(),
```
Comment (AbbasJay) [inline lib/db/schema.ts]: @hyperleash you can remove these now that my branch is merged

Comment (marat-davudov) [reply lib/db/schema.ts]: Done

**Pull request merged**